### PR TITLE
Update functions.rst

### PR DESCRIPTION
**Pull request by megan201296**
Add examples that may be useful to new users of pipelines.

### CODE REVIEW

File: pages/pipelines/functions.rst [hunk 1]
```diff
@@ -322,6 +334,11 @@ contains
 
 Checks if ``value`` contains ``search``, optionally ignoring the case of the search pattern.
 
+Example::
+
+        // Check if the `computer.corp.net` is in the `hostname` field. Ignore case.
+        contains(to_string($message.hostname), "computer.corp.net", true)
```
Comment (jalogisch) [inline pages/pipelines/functions.rst]: could you please change that to `example.org` as we use that all over the documentation.

File: pages/pipelines/functions.rst [hunk 2]
```diff
@@ -508,6 +525,11 @@ processed for this message.
 
 If ``remove_from_default`` is ``true``, the message is also removed from the default stream "All messages".
 
+Example::
+
+        // Route the current processed message to a stream named `Custom Stream`
+        route_to_stream("Custom Stream");
```
Comment (jalogisch) [inline pages/pipelines/functions.rst]: using the name is onyl the second best option - could you also add example for the route to a streamid.

From a performance perspective this is prefered and in addition the name can be assigned multiple times. The ID is uniq.

**Review by jalogisch — CHANGES_REQUESTED** [review]
thank you for this little update. We need to improve this part of the documentation and giving some samples is a nice to have.

Could you please check on my comments.

File: pages/pipelines/functions.rst (hunk 1 above)
Comment (megan201296) [reply pages/pipelines/functions.rst]: Done!

File: pages/pipelines/functions.rst (hunk 2 above)
Comment (megan201296) [reply pages/pipelines/functions.rst]: Done. I've never actually used the ID. I believe that it goes in quotes, is that correct? Add it that way but feel free to fix if incorrect.

**Comment by megan201296** [comment]
Made the requested changes @jalogisch! Thanks for the feedback.

**Review by jalogisch — APPROVED** [review]

**Comment by jalogisch** [comment]
thank you for the contribution @megan201296

**Pull request merged**